crates/capi/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "wasmtime-capi"
+version = "0.9.0"
+authors = ["The Wasmtime Project Developers"]
+description = "C API to expose the Wasmtime runtime"
+license = "Apache-2.0 WITH LLVM-exception"
+repository = "https://github.com/bytecodealliance/wasmtime"
+readme = "README.md"
+edition = "2018"
+publish = false
+
+[lib]
+name = "wasmtime"
+crate-type = ["staticlib", "cdylib"]
+doc = false
```

Comment:
I'm curious why `rustdoc = false` here. The docs are obviously less interesting than in a regular Rust API crate, but they're still occasionally useful to have something to refer to, specially since the official wasm-c-api repo doesn't have any documentation either.

Reply:
The name `wasmtime` here conflicts with the `wasmtime` crate itself, so `cargo doc --all` will fail (as is currently executed on CI). We could also change it to `cargo doc --exclude`, though, and I think it would work as well.

Reply:
Oh, hmm. That would mean anyone doing `cargo doc --all` would have to use `--exclude`, which seems unfortunate. Maybe it's best to just use `doc = false` here for now.